tests/phpunit/AutoReview/Makefile/MakefileTest.php

```diff
@@ -187,37 +184,6 @@ public static function rootTestTargetProvider(): iterable
         ];
     }
 
-    public function test_all_docker_test_targets_are_properly_declared(): void
```

Comment:
Looking for ~10 minutes to this test and the second one that is removed, I didn't get it and it looks like it works incorrectly making incorrect assumption regarding targets naming.

@theofidry do you have any idea what does it do and why it fails with the current Makefile?

Reply:
I'll comment in each tests

Reply:
This test ensures that we follow the pattern that for each phony target `test-*-docker`, it contains all `test-*-*-docker`.

For instance:

- `test-docker` contains `test-unit-docker`, `test-e2e-docker` and `test-infection-docker`.
- `test-unit-docker` contains `test-unit-82-docker`, `test-unit-83-docker`, etc.